Add references to Content guide alongside Style guide

aimeeu opened the pull request:
Add references to newly added Content guide alongside Style guide references. I forgot a few places on a few pages in the original Content guide PR.

Fixes #16100 

Preview: https://deploy-preview-16101--kubernetes-io-master-staging.netlify.com/docs/contribute/

aimeeu commented:
/assign @steveperry-53

aimeeu commented:
FYI - I changed the bullet lists to numbered lists in the Content Guide (outside the scope of this PR I know). Contributors will find it easier to refer to points in the numbered list.

aimeeu commented on content/en/docs/contribute/intermediate.md:
```diff
@@ -106,8 +106,8 @@ more information about the responsibilities of reviewers and approvers, see
-  SIG Docs guidelines, such as the
-  [style guide](/docs/contribute/style/style-guide). Only people listed as
+  SIG Docs guidelines found in the [Content](/docs/contribute/style/content-guide) and 
+  [Style](/docs/contribute/style/style-guide) guides. Only people listed as
```
@kbhawkey done

kbhawkey commented on content/en/docs/contribute/style/content-guide.md:
```diff
@@ -45,68 +45,68 @@ effort from project maintainers and tends to become outdated more quickly.
-        - Adding a tutorial on how to use a CNCF project or a project in the kubernetes or kubnetes-sigs GitHub organizations if the project has its own documentation
+    1. Allowed:
+        1. Referring to or linking to existing documentation about a CNCF project or a project in the kubernetes or kubernetes-sigs GitHub organizations
+            1. Example: for installating Kubernetes in a learning environment, including a prerequisite stating that successful installation and configuration of minikube is required and linking to the relevant minikube documentation
```
nit: line 65, spelling fix, `installating` 😃
nit: line 54, start as `Installing Kubernetes ...` or list the three items (?), add a comma after required?

aimeeu replied:
This will be done in a separate PR.  I've reverted my content-guide.md changes.

kbhawkey commented on content/en/docs/contribute/style/content-guide.md:
```diff
@@ -45,68 +45,68 @@ effort from project maintainers and tends to become outdated more quickly.
+        1. Referring to or linking to existing documentation about a CNCF project or a project in the kubernetes or kubernetes-sigs GitHub organizations
+            1. Example: for installating Kubernetes in a learning environment, including a prerequisite stating that successful installation and configuration of minikube is required and linking to the relevant minikube documentation
+        1. Adding content for a kubernetes or kubernetes-sigs project that doesn't have its own instructional documentation; before adding, 
+            1. Example: including [kubadm](https://github.com/kubernetes/kubeadm) installation and troubleshooting instructions because the kubeadm project does not have its own documentation in github or on a separate website   
```
nit: spelling `kubeadm`

aimeeu replied:
This will be done in a separate PR.  I've reverted my content-guide.md changes.

kbhawkey commented on content/en/docs/contribute/style/content-guide.md:
```diff
@@ -45,68 +45,68 @@ effort from project maintainers and tends to become outdated more quickly.
+    1. If yes, then:
+        1. Does the project have its own documentation?
+            1. if yes, link to the project's documention from the Kubernetes documentation 
+            1. if no, add the content to the project's repository if possible and then link to it from the Kubernetes documentation 
```
nit: line 51, extra space at end; items start with lower case: `If yes`

aimeeu replied:
This will be done in a separate PR.  I've reverted my content-guide.md changes.

kbhawkey commented on content/en/docs/contribute/style/content-guide.md:
```diff
@@ -45,68 +45,68 @@ effort from project maintainers and tends to become outdated more quickly.
+            1. if yes, link to the project's documention from the Kubernetes documentation 
+            1. if no, add the content to the project's repository if possible and then link to it from the Kubernetes documentation 
+    1. If no, then:
+        1. Stop!
```
nit: Not sure `Stop!` is needed

aimeeu replied:
This will be done in a separate PR.  I've reverted my content-guide.md changes.

kbhawkey commented on content/en/docs/contribute/style/content-guide.md:
```diff
@@ -45,68 +45,68 @@ effort from project maintainers and tends to become outdated more quickly.
+        1. Adding a brief introductory paragraph about a CNCF project or a project in the kubernetes or kubernetes-sigs GitHub organizations; the paragraph may contain links to the project        
+    1. Not Allowed:    
+        1. Adding content describing a vendor-specific product
+        1. Adding content describing an open source project that is not a CNCF project or a project in the kubernetes or kubnetes-sigs GitHub organizations
```
nit: line 87, spelling kubnetes-sigs
The list of accepted projects (CNCF, kubernetes, kubernetes-sigs Github organizations) is a lot to parse. Is there any way to make a key or acronym?

aimeeu replied:
This will be done in a separate PR.  I've reverted my content-guide.md changes.

kbhawkey commented on content/en/docs/contribute/style/content-guide.md:
```diff
@@ -45,68 +45,68 @@ effort from project maintainers and tends to become outdated more quickly.
-        - Linking to vendor-specific tutorials regardless of the training provider
-            - Example: linking to Linux Academy courses such as [Google Kubernetes Engine Deep Dive](https://linuxacademy.com/google-cloud-platform/training/course/name/google-kubernetes-engine-deep-dive) and [Amazon EKS Deep Dive](https://linuxacademy.com/course/amazon-eks-deep-dive/)
+    1. Allowed:    
+        1. Linking to vendor-neutral Kubernetes training courses offered by the [CNCF](https://www.cncf.io/), the [Linux Foundation](https://www.linuxfoundation.org/), and the [Linux Academy](https://linuxacademy.com/), which is a partner of the Linux Foundation
```
nit: Which Linux Academy courses are allowed? It is confusing that you provide Linux Academy examples below that are not allowed

aimeeu replied:
This will be done in a separate PR.  I've reverted my content-guide.md changes.

kbhawkey commented:
content guide page preview:
https://deploy-preview-16101--kubernetes-io-master-staging.netlify.com/docs/contribute/style/content-guide/

aimeeu commented:
@kbhawkey I appreciate your comments. I should not have renumbered the content guide as part of this PR - out of scope. I've reverted my content-guide.md changes and added a comment to Issue #16143 to address your comments on the PR when I push a PR for that issue.

kbhawkey commented:
hello @aimeeu. I went through this page again; forgive the nit picky comments. In general, there are a lot of ideas/rules to check when adding content (almost too many rules and derivations). The workflows seems overly complicated. I also considered the current k8s docs.
One thing that may help visually is to create sub-headings for the main categories, instead of the nested flowchart. This will create links to the individual sections and breakup the chart.

aimeeu commented:
Yes that was my thinking exactly! I just haven't brain-dumped to anyone else.

> One thing that may help visually is to create sub-headings for the main categories, instead of the nested flowchart. This will create links to the individual sections and breakup the chart.

aimeeu commented:
@jaypipes or @kbhawkey - do either of you have time to review and LGTM?

jaypipes commented:
Apologies for the delay @aimeeu  
/lgtm

aimeeu commented:
@steveperry-53 this now has an LGTM - would you approve when you have a minute? Thanks!

makoscafee commented:
/approve
